src/utils/vc/credentials.js

```diff
@@ -250,12 +321,9 @@ export async function verifyCredential(credential, {
  *   `@context` (if it is not present, it's added to the context list).
  * @return {Promise<object>} The signed credential object.
  */
-export async function issueCredential(keyDoc, credential, compactProof = true, documentLoader = null, purpose = null, expansionMap = null, issuerObject = null, addSuiteContext = false, useProofValue = false) {
-  // Get suite from keyDoc parameter
-  const suite = getSuiteFromKeyDoc(keyDoc, useProofValue);
-  if (!suite.verificationMethod) {
-    throw new TypeError('"suite.verificationMethod" property is required.');
-  }
+export async function issueCredential(keyDoc, credential, compactProof = true, documentLoader = null, purpose = null, expansionMap = null, issuerObject = null, addSuiteContext = true, type = 'jsonld') {
```

Comment:
What are the possible values of `type`? Its default value is `jsonld` but inside the function, it checks for only 2 values `proofValue` and `jwt` which suggests that the default value should be one of those

Reply:
well, logically it suggests that jsonld is the default type and others are checked incase. it can be all 3 of them - ill make them into constants and add some documentation strings

Reply:
Thanks. An enum-like type for `type` will help

Reply:
this was updated